kapitan/resources.py

```diff
@@ -340,16 +338,14 @@ def get_inventory(inventory_path, ignore_class_not_found: bool = False) -> Inven
     backend = get_inventory_backend(backend_id)
 
     logger.debug(f"Using {backend.__name__} as inventory backend")
-
     try:
         inventory_backend = backend(
             inventory_path=inventory_path,
             compose_target_name=compose_target_name,
             ignore_class_not_found=ignore_class_not_found,
         )
     except InventoryError as e:
-        logger.fatal(e)
-        raise
+        sys.exit(1)
```

Comment:
Some people are using kapitan as a library, and this would break their workflows. 

let's find a better way to handle it, I was trying to sort out this mess as well so ok with the overall intention

Reply:
but there was already a sys.exit(1) on line 316, I just moved it to another place. 
How was this working for people using it as a library with the existing sys.exit?

Reply:
I see. Ok, I will remove the sys.exit as part of another effort another time